client/src/pages/pot-instance/components/LandingPage/LandingPage.tsx

```diff
@@ -1,20 +1,75 @@
-import { Button } from '@mui/material';
-import { FC } from 'react';
+import * as React from 'react';
+import { Button, ToggleButtonGroup } from '@mui/material';
+import MuiToggleButton from '@mui/material/ToggleButton';
+import { styled, useTheme } from '@mui/material/styles';
+import { FC, useState } from 'react';
 import styles from './styles.module.css';
-
 // https://pngtree.com/free-backgrounds-photos/spicy-hot-pot
 
 interface Props {
-  updatePotType: (type: number) => void;
-  startHotPot: () => void;
+  startHotPot: (soupbase: string[]) => void;
   hotpotStart: boolean;
 }
 
-const LandingPage: FC<Props> = ({
-  updatePotType,
-  startHotPot,
-  hotpotStart
-}) => {
+const LandingPage: FC<Props> = ({ startHotPot, hotpotStart }) => {
+  const [potType, setPotType] = useState('1');
+  const [soupbase, setSoupbase] = useState(['bone']);
+
+  // below is required to use custom mui themes
+  const theme = useTheme();
+
+  // function to update soupbase selection based on the number of pots selected
+  const handlePotChange = (newAlignment: string) => {
+    // queue for soupbase based on potType number
+    const soupbaseTemp = soupbase;
+    let i = soupbaseTemp.length;
+    while (i > parseInt(newAlignment, 10)) {
+      soupbaseTemp.shift();
+      i -= 1;
+    }
+    setSoupbase([...soupbaseTemp]);
+  };
+
+  // updates the toggle buttons for pot type. enforces that one is always toggled
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    newAlignment: string
+  ) => {
+    if (newAlignment !== null) {
+      setPotType(newAlignment);
+      handlePotChange(newAlignment);
+    }
```

Comment:
These !== null clauses are to guard against an inherent behavior for mui togglebuttons. This just prevents a button from being de-selected

Reply:
Sounds good, I would recommend also adding this as a comment.

Reply:
added!